ingestion_tools/scripts/standardize_dirs.py

```diff
@@ -126,7 +62,8 @@ def do_import(config, tree, to_import, metadata_import, to_iterate, kwargs, pare
             if filter_patterns and not list(filter(lambda x: x.match(item.name), filter_patterns)):
                 print(f"Skipping {item.name}..")
                 continue
-            if import_class in to_import:
+            # If item is to be imported, only import it if the source data allows import
+            if import_class in to_import and item.allow_imports:
```

Comment:
I think having the `allow_imports` checks in the `import_item()` and `import_metadata()` methods makes it harder to accidentally import data when we shouldn't. Relying on the caller to check this flag seems pretty insecure to me.